package.json

```diff
@@ -21,12 +21,13 @@
     "react-draft-wysiwyg": "^1.13.2",
     "react-ga": "^2.6.0",
     "react-lazy-load": "^3.0.13",
+    "react-router-dom": "^5.1.2",
     "react-select": "^3.0.4",
     "styled-components": "^4.3.2"
   },
   "scripts": {
     "lint": "eslint \"./{pages,components}/**/*.js\" \"*.js\"",
-    "dev": "node server/index.js",
+    "dev": "nodemon server/index.js",
```

Comment:
If you make this change, ensure that `nodemon` is installed in the `package.json` as a developer dependency.